src/ng_link/exaspim_link.py

```diff
@@ -9,14 +9,14 @@
 
 
 def generate_exaspim_link(
-        xml_path: Optional[str] = None,
-        s3_path: str = None,
-        vmin: float = 0,
-        vmax: float = 200,
-        opacity: float = 1.0,
-        blend: str = "default",
-        output_json_path: str = ".",
-        dataset_name: Optional[str] = None,
+    xml_path: Optional[str] = None,
+    s3_path: str = None,
+    vmin: float = 0,
+    vmax: float = 200,
```

Comment:
I know this comes from an earlier PR but could you please the Optional typing to s3_path, vmin, vmax, etc parameters that don't have the optional? e.g., vmin: Optional[float] = 0.0

Reply:
Since they all have defaults, I'll change all of them to Optional